release-notes/latest.md

```diff
@@ -0,0 +1,6 @@
+#### <sub><sup><a name="v551-note-1" href="#v551-note-1">:link:</a></sup></sub> fix
```

Comment:
This is fine, but so you're aware these IDs can be anything unique, so the PR number or issue number is a natural choice, i.e.
```suggestion
#### <sub><sup><a name="v551-note-4326" href="#v551-note-4326">:link:</a></sup></sub> fix
```